### PR TITLE
Add Packaging Docs

### PR DESCRIPTION
**Pull request by TheBrokenRail**

### CODE REVIEW

File: src/contributing/packaging.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Packaging
+
+**NOTE:** This is not extensivly tested, and may break
+
+## Debian
+
+To package Darling for Debian-baded system we provide the ```tools/makedeb``` script.
```
Comment (bugaevc) [inline src/contributing/packaging.md:7]: Inline code should use single backticks, triple backticks are for code blocks.

Comment (TheBrokenRail) [reply src/contributing/packaging.md]: Fixed

**Comment by TheBrokenRail** [comment]
Any updates on this?

**Comment by TheBrokenRail** [comment]
@bugaevc

**Pull request merged**